### PR TITLE
Likes remote objects

### PR DESCRIPTION
**Pull request by HailanXyouknow**
To test the code you can do the following in postman:

POST http://127.0.0.1:8000/service/author/73f4af3d-7c9f-4ae2-bd14-e0acbce8fe8c/inbox/

```
 {
     "type": "like",
     "author": {
        "username": "username",
        "type": "author",
        "id": "c4dba4ce-7554-40f9-acf4-df8e2dc297e1",
        "host": "https://konnection-server.herokuapp.com/",
        "displayName": "Sent from Hailan's Code",
        "url": "https://konnection-client.herokuapp.com/author/c4dba4ce-7554-40f9-acf4-df8e2dc297e1",
        "github": ""
    },
     "object":"https://cyver.herokuapp.com/api/author/73f4af3d-7c9f-4ae2-bd14-e0acbce8fe8c/posts/aeaa5be4-8bea-4370-8fbd-b02d1eae8c12"
}
```

checkout inbox at Justin's team server : https://cyver.herokuapp.com/api/author/73f4af3d-7c9f-4ae2-bd14-e0acbce8fe8c/inbox

Justin team's frontend: https://qonnectionz.netlify.app/

### CODE REVIEW

File: server/inbox/views.py [hunk 1]
```diff
@@ -80,6 +83,31 @@ def post(self, request, *args, **kwargs):
...
+            else:
+                try:
+                    remote_server = Node.objects.get(remote_server_url=object_host)
+                except Node.DoesNotExist:
+                    return Response({'error':'Could not find remote server user'}, status=status.HTTP_404_NOT_FOUND)
+
+                headers = {'content-type': 'application/json'}
+                r = requests.post(
+                    f"{object_host}api/author/{request_author_id}/inbox/",
+                    data=json.dumps(request.data),
```
Comment (keanweng97) [inline server/inbox/views.py:104]: you can just use json=json_data here for POSTing JSON, headers will be automatically added

**Review by keanweng97 — COMMENTED** [review]
other than that, LGTM ✨

File: server/inbox/views.py (hunk 1 above)
Comment (HailanXyouknow) [reply server/inbox/views.py:104]: ohhhhhh, nice nice. Thanks, Kean, I'll change that now.

**Review by keanweng97 — APPROVED** [review]
LGTM ✨✨✨✨✨

**Pull request merged**